src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form">
```

Comment:
Add empty lines between sibling elements like:
```suggestion
        <legend>Personal information</legend>
        
        <div class="form">
```